platform/Keys.roc

```diff
@@ -16,32 +16,38 @@ import InternalKeyboard
 
 KeyboardKey : InternalKeyboard.KeyboardKey
 
-Keys : Dict InternalKeyboard.KeyboardKey InternalKeyboard.KeyState
+Keys : InternalKeyboard.Keys
 
 down : Keys, KeyboardKey -> Bool
 down = \keys, key ->
-    state = Dict.get keys key
-    state == Ok Down || state == Ok PressedRepeat
```

Comment:
I lost a bunch of time trying to figure out why `Ok Down` != `Down`, so now I think a Roc linter should advise against using ``==` with tags vs matching